src/main/java/com/company/zayats/hw18/Xml.java

```diff
@@ -0,0 +1,78 @@
+package com.company.zayats.hw18;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Xml {
```

Comment:
Can be combined with the JSON class, they have the same code, only the regular expressions are different